fmoo/react-typeahead
# Preserve url hash

### What does it do?
Fixes the behavior of changing the behavior of altering url hash when options are clicked.

### Where should the reviewer start?
This is a minor change `src/typeahead/option.js`

### How should this be manually tested?
Click directly on an `<a>` tag in the options list, notice that the url hasn't changed.
